static/styles/subscriptions.css

```diff
@@ -882,50 +869,6 @@
         margin-bottom: 5px;
     }
 
-    .editable-section {
-        position: relative;
-        display: inline;
-        min-width: 10px;
-        max-width: 100%;
-
-        &[contenteditable="true"] {
-            display: inline-block;
-            color: hsl(170, 48%, 54%);
-
-            &:focus {
-                outline: none;
-            }
-        }
-    }
-
-    .editable {
-        position: relative;
-        top: -1px;
-        margin-left: 5px;
-        color: hsl(0, 0%, 67%);
-        cursor: pointer;
-        font-size: 1.4rem;
-        font-weight: 300;
-
-        transition: all 0.3s ease;
-
-        &:empty::before {
-            content: "\f040";
-            font-family: FontAwesome;
-            font-size: 0.8rem;
-            font-weight: normal;
-        }
-
-        &:not(:empty) {
-            position: relative;
-            top: 3px;
-        }
-
-        &:hover {
-            color: hsl(0, 0%, 27%);
-        }
-    }
```

Comment:
How carefully have you audited that we found all of the old `editable` CSS?

Reply:
I used `git grep -n "editable\|contenteditable" static/styles/` and after this removal we only have `contenteditable` related css for settings pages where input pills are used like user grorup settings modal.